### DISCUSSION TITLE
Delete Excluded Objects

### DISCUSSION THREAD

**Discussion by arbennet** [Azure-Player/azure.datafactory.tools#146] [Q&A]
Hi Kamil,

I'd like to delete objects that have been excluded from deployment.  I've read the documentation and tried many configurations.

I've defined my exclusion as:
```-pipeline.PL_Ingestion_Controller```

I can see that this is exclusion is working properly in the log:
![image](https://user-images.githubusercontent.com/60998792/135335380-1f51dc59-19fd-4bb7-bdf2-46aac21183b1.png)

As far as I can tell, I've configured these exclusions to be deleted:
![image](https://user-images.githubusercontent.com/60998792/135335676-d212c410-932d-47f3-83b3-d887ab2f01f2.png)
![image](https://user-images.githubusercontent.com/60998792/135335713-5e1bea2e-dc9e-48d9-8814-13893ed278dd.png)

However, the pipeline is still there after release!

![image](https://user-images.githubusercontent.com/60998792/135335877-59533c73-56e9-44c3-b248-e97ce82901c5.png)
![image](https://user-images.githubusercontent.com/60998792/135335932-7f823fc8-90c9-4482-9c83-42986bdd9e77.png)

What am I doing wrong here?

**Comment by NowinskiK** [comment]
hmmm... all looks good TBH. 
Can you check this with PS module and attach here a script and related files that I can check?
